packages/location-field/src/utils.ts

```diff
@@ -64,3 +70,31 @@ export const getCombinedLabel = (properties: Properties): string => {
   }
   return properties?.label || "";
 };
+
+/**
+ * Helper method to append text in parentheses to some other text,
+ * if the added text is not null or blank.
+ */
+export const addInParentheses = (
+  mainText: string,
+  extraText?: string
+): string => {
+  return extraText && extraText !== ""
+    ? `${mainText} (${extraText})`
```

Comment:
Is this i18n compliant? What about RTL languages?

Reply:
It looks like the direction of parentheses should apply automatically depending on the direction of the language (see https://www.w3.org/International/articles/inline-bidi-markup/#mirrored), but we may need extra work if mixing LTR and RTL directions (e.g. RTL place names  and English addresses).

Reply:
Additional info for parentheses:
- Chinese has [extra spacing outside the parentheses](https://en.wikipedia.org/wiki/Chinese_punctuation#Marks_similar_to_European_punctuation)
- [Hebrew uses the regular parentheses](https://en.wikipedia.org/wiki/Hebrew_punctuation#Brackets/parentheses) (see comment above for mirroring)
- [Arabic folks seem to use regular parentheses](https://forum.wordreference.com/threads/arabic-parentheses.157788/) too for non-religious documents.
- Korean: there is typically no space before parentheses.

Reply:
This item is addressed in https://github.com/opentripplanner/otp-ui/pull/533/commits/332726cbc87887218c61f296c7639ab2e09ff403. Thanks for pointing it out.